### PR TITLE
Make some fixes according to Go Report Card

### PR DESCRIPTION
**Pull request by n10v**

### CODE REVIEW

File: bloomfilter/bloom.go [hunk 1]
```diff
@@ -2,7 +2,6 @@ package olilib
 
 import (
 	"fmt"
-	"github.com/GrappigPanda/Olivia/lru_cache"
```
Comment (Ianleeclark) [inline bloomfilter/bloom.go:5]: It looks like this removed line is causing a build failure.

**Comment by n10v** [comment]
I wil fix now

File: dht/peer.go [hunk 2]
```diff
@@ -4,15 +4,14 @@ import (
 	"crypto/md5"
 	"encoding/hex"
 	"fmt"
-	"github.com/GrappigPanda/Olivia/bloomfilter"
```
Comment (Ianleeclark) [inline dht/peer.go:7]: This still needs to be imported

File: dht/peer.go [hunk 3]
```diff
@@ -4,15 +4,14 @@ import (
...
-	"github.com/GrappigPanda/Olivia/config"
-	"github.com/GrappigPanda/Olivia/network/message_handler"
-	"github.com/GrappigPanda/Olivia/network/receiver"
```
Comment (Ianleeclark) [inline dht/peer.go:10]: And the receiver as well.

File: network/incoming/incoming_network.go [hunk 4]
```diff
@@ -3,14 +3,14 @@ package incomingNetwork
 import (
 	"bufio"
 	"fmt"
-	"github.com/GrappigPanda/Olivia/bloomfilter"
```
Comment (Ianleeclark) [inline network/incoming/incoming_network.go:6]: This needs to be imported.

File: network/incoming/incoming_network_test.go [hunk 5]
```diff
@@ -3,16 +3,16 @@ package incomingNetwork
 import (
 	"bufio"
 	"fmt"
-	"github.com/GrappigPanda/Olivia/bloomfilter"
```
Comment (Ianleeclark) [inline network/incoming/incoming_network_test.go:6]: Keep this import

File: network/incoming/language_map_test.go [hunk 6]
```diff
@@ -1,12 +1,12 @@
 package incomingNetwork
 
 import (
-	"github.com/GrappigPanda/Olivia/bloomfilter"
```
Comment (Ianleeclark) [inline network/incoming/language_map_test.go:4]: Keep this import

**Comment by Ianleeclark** [comment]
All right @bogem, I think we've done it. I just marked all the ones that need to stay being imported.

**Pull request closed without merging**